### DISCUSSION TITLE
Is PyG affected by torchtriton malicious dependency?

### DISCUSSION THREAD

**Discussion by victormaricato** [pyg-team/pytorch_geometric#6337] [Q&A] [answered]
Recently, [PyTorch disclosed a malicious dependency chain](https://pytorch.org/blog/compromised-nightly-dependency/) related to `torchtriton`, now renamed `pytorch-triton.`

Is/Was PyG affected by this malicious dependency?

**Comment by rusty1s** [comment, marked as answer]
No, `torchtriton` is not a dependency of PyG.
